README.md

```diff
@@ -38,6 +38,31 @@ cluster on AWS.
 
 ------
 
+## Compatibility with Cluster API and Kubernetes Versions
+
+This provider's versions are compatible with the following versions of Cluster API:
+
+||Cluster API 0.1|
+|-|-|
+|AWS Provider 0.1|✓|
```

Comment:
This should probably be 0.2, since there are already a v0.1.0 and a v0.1.1 releases targeting pre-v1alpha1 snapshots of cluster-api